packages/react/test/unit/useRevisions.test.tsx

```
@@ -0,0 +1,338 @@
+        >
+          <TestComponent />
+        </DocumentProvider>,
+      );
+
+      await waitFor(() => {
+        expect(hookResult).toBeDefined();
+      });
+
+      await expect(
+        hookResult!.createRevision('v1.0'),
+      ).rejects.toThrow('Client or document is not ready');
```

ESLint

[error] Replace `⏎········hookResult!.createRevision('v1.0'),⏎······).rejects.toThrow('Client·or·document·is·not·ready'` with `hookResult!.createRevision('v1.0')).rejects.toThrow(⏎········'Client·or·document·is·not·ready',⏎······`

(prettier/prettier)